src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <section class="card" data-qa="card">
+      <div class="card__img-postion">
+        <img class="card__img"
+          src="./images/imac.jpeg"
+          alt="Apple A1419 iMac 27 diagonal Retina 5K Monoblock"
+        >
+      </div>
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
```

Comment:
it's better not add `<br>` and use CSS for adjusting the styles
```suggestion
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
```